src/torrent.cpp

```diff
@@ -5726,9 +5735,6 @@ namespace {
 
 		if (is_seed()) return;
 
-		// setting file priority on a torrent that doesn't have metadata yet is
-		// similar to having passed in file priorities through add_torrent_params.
-		// we store the priorities in m_file_priority until we get the metadata
```

Comment:
I think this comment is useful. it should probably be moved down to the assignment to `m_file_priority` in the else-block. But I would prefer to do that in master to make this simpler to merge

Reply:
Agreed.